server/utils/searchEformValidation.ts

```diff
@@ -50,6 +55,10 @@ const schema = Joi.object({
       'date.format': 'Submission date to must be a valid date',
     })
     .custom((value, helpers) => {
+      // Skip validation if ENVIRONMENT_NAME is 'archive'
+      if (config.environmentName === 'archive') {
```

Comment:
Similar here, 'missing start date' and 'end date earlier' validations are skipped in the archive environment.